public/javascripts/application.js

```diff
@@ -1,2 +1,18 @@
-// Javascript Goes Here~
-//
+(document).ready(function(){
+var person = function() {
+
```

Comment:
```javascript
var first = 'Crab';
var last = 'Daddy';
```

This would have illustrated that the variables are not accessible unless returned by the function in the object literal.